docs/tutorials/basics/part4.md

```diff
@@ -106,7 +106,7 @@ curl -L -X PUT 'http://localhost:8080/v0/connectors/part-datagen' \
           { "limit": 3,
             "fields": {
               "id": { "strategy": { "name": "increment" }, "range": [1, 4] },
```

Comment:
I was hoping we'd be able to write `"strategy": "increment"` without the extra nesting.